source/renderer/app/components/wallet/mnemonic-input/MnemonicAutocompleteLayout.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { ClipboardEventHandler, RefObject, useCallback } from 'react';
+import cx from 'classnames';
+import { FormField } from 'react-polymorph/lib/components/FormField';
+import { Options } from 'react-polymorph/lib/components/Options';
+import { OptionsSkin } from 'react-polymorph/lib/skins/simple/OptionsSkin';
+import * as styles from './MnemonicAutocompleteLayout.scss';
+
+interface MnemonicInputSkinProps {
+  onClick: () => void;
+  onChange: (value: string) => void;
+  onSelect: (value: string) => void;
+  onBlur: () => void;
+  onPaste: ClipboardEventHandler<HTMLInputElement>;
+  toggleMouseLocation: () => void;
+  toggleOpen: () => void;
+
+  value: string;
+  ordinalNumber: number;
+  disabled: boolean;
```

Comment:
We should make sure that this component will not trigger any callbacks when it is disabled (to remove the `useCallback` + checks in the container)